README.md

```diff
@@ -25,10 +25,22 @@ npm run coverage
 ```
 
 ## Deployment
-```
-npm run deployOP
-npm run deployFTM
-```
+1. In ```000_deploy_dest_chain.ts```, fill in the ```veIB``` address
```

Comment:
should put veIB as `namedAccounts` in `hardhat.config.ts`

example: https://github.com/ibdotxyz/ib-deployment/blob/op/hardhat.config.ts

Reply:
done